setup.cfg

```diff
@@ -1,30 +1,36 @@
 [metadata]
 name = pidgy
-author="deathbeds"
-author_email=tony.fast@gmail.com
-url=https://github.com/deathbeds/pidgy
+author = "deathbeds"
+author_email = tony.fast@gmail.com
+url = https://github.com/deathbeds/pidgy
 description = literate programming in ipython
 long_description = file: readme.md
 long_description_type = text/markdown
 keywords = markdown, jupyter
 license = BSD 3-Clause License
 classifiers =
+    Framework :: Jupyter
+    Intended Audience :: Developers
+    Intended Audience :: Science/Research
     License :: OSI Approved :: BSD License
     Programming Language :: Python :: 3
-    
+    Topic :: Text Processing :: Markup :: HTML
+
 [options]
 zip_safe = False
 include_package_data = True
 install_requires =
-    markdown-it-py[linkify,plugins]
     htmlmin
     jinja2
+    linkify-it-py
```

Comment:
like many things, `micropip` (and by extension `piplite`) don't work very well with extras